packages/cozy-harvest-lib/src/models/ConnectionFlow.spec.js

```diff
@@ -571,64 +501,91 @@ describe('ConnectionFlow', () => {
       )
     })
 
-    it('should return unmodified account trigger folder does not exist', async () => {
+    it('should return unmodified account trigger when folder does not exist', async () => {
       const { flow, client } = setup()
       prepareTriggerAccount.mockResolvedValue(fixtures.updatedAccount)
 
       ensureTrigger.mockResolvedValue(
         fixtures.createdTriggerWithFolder.attributes
       )
       client.query.mockRejectedValue(new Error('404'))
-      await flow.ensureTriggerAndLaunch(client, {
+      await setupSubmit(flow, {
         account: fixtures.existingAccount,
         trigger: fixtures.existingTrigger,
         konnector: fixtures.konnectorWithFolder
       })
       expect(flow.account).toEqual(fixtures.updatedAccount)
-      expect(saveAccount).not.toHaveBeenCalled()
+      expect(saveAccount).toHaveBeenCalledTimes(1)
```

Comment:
The change is that we call flow.handleFormSubmit in this test instead of ensureTriggerAndLaunch. flow.handleFormSubmit calls createOrUpdateAccount, which is why a call to saveAccount is expected.